cluster/addons/dns/skydns-rc.yaml.in

```diff
@@ -28,12 +28,17 @@ spec:
             memory: 50Mi
         command:
         - /usr/local/bin/etcd
+        - -data-dir
+        - /var/etcd/data
```

Comment:
I have now mounted mounted emptydir on /var/etcd/data 
I used that as a mount mount for consistency. Our other etcd container on master uses /var/etcd/data